src/jvmMain/kotlin/com/github/doyaaaaaken/kotlincsv/client/CsvWriter.kt

```diff
@@ -144,7 +144,7 @@ actual class CsvWriter actual constructor(
     /**
      * write all rows to string
      */
-    fun writeAll(rows: List<List<Any?>>): String {
-        return writeString { writeRows(rows) }
+    fun writeAllAsString(rows: List<List<Any?>>): String {
```

Comment:
Rename this method into `writeAllAsString` because there are many other `writeAll` methods and their behavior is slightly different from the behavior of this method.